python_projects/Kickstarter/widgety_test.ipynb

```diff
@@ -0,0 +1,158 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": 30,
+   "id": "07804aa2",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import pandas as pd\n",
+    "import matplotlib as plt\n",
+    "import seaborn as sns\n",
+    "\n",
+    "import ipywidgets as widgets\n",
+    "from ipywidgets import interact"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "id": "c28ef81f",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "#funkcja import pliku\n",
+    "def f_file_import(name:str):\n",
```

Comment:
może fajnie byłoby wrzucić tutaj separator jako argument (np. z przecinkiem jako domyślnym)? To się jednak bardzo często różni w zależności od pliku